rollup.config.js

```diff
@@ -11,54 +9,23 @@ const output = (name, format) => ({
   sourcemap: true,
   globals: {
     '@webex/common': '@webex.common',
-    bufferutil: 'bufferutil',
-    'prop-types': 'PropTypes',
-    react: 'React',
     rxjs: 'rxjs',
     'rxjs/operators': 'rxjs.operators',
-    'react-dom': 'ReactDOM',
-    'spawn-sync': 'spawnSync',
-    'utf-8-validate': 'utf8Validate',
+    webex: 'webex',
   },
 });
 
 export default [
   {
     input: 'src/index.js',
-    output: [
-      output('webexSDKComponentAdapter', 'cjs'),
```

Comment:
Removing this will break the package.json's `"main": "dist/webexSDKComponentAdapter.cjs.js",`